tests/integration.test.js

```diff
@@ -0,0 +1,175 @@
+const { expect, it, describe } = require('@jest/globals')
+const { v4: uuidv4 } = require('uuid')
+
+const helpers = require('./gh-api-helpers')
+
+const RUNNER_COUNT = 3
+const TIMEOUT = 300000
+const BUSTED_EVENTS_FILE_PREFIX = 'busted-events-'
+
+const aggregateBustedEvents = (artifacts) => {
+  const bustedEvents = []
+
+  for (const [key, value] of Object.entries(artifacts)) {
+    if (key.startsWith(BUSTED_EVENTS_FILE_PREFIX)) {
+      const runnerID = key.split(BUSTED_EVENTS_FILE_PREFIX)[1]
+
+      const events = Object.values(value[0])[0].split('\n')
+      for (e of events) {
+        try {
+          e = JSON.parse(e)
+        } catch (err) {
```

Comment:
Why is this error caught?  Is there a legitimate reason why the events file could contain unparseable JSON?

Reply:
no I was being lazy and (temporarily) postponed debugging why some events were empty strings, fixed now.